addOns/sequence/CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Added
-- Initial sequence-activeScan implementation.
+- Initial sequence-activeScan implementation
+- Default Sequence active scan policy.
```

Comment:
Technically it's not the default, it's not being used (by default) anywhere.

Reply:
Rather than change the changelog entry it should be changed the code to use it by default (IMO).

Reply:
This will be handled as follow up.